gh-pages-stub/public/js/search.js

```diff
@@ -0,0 +1,93 @@
+(function() {
+
+  function isBlank(str) {
+    return (!str || /^\s*$/.test(str));
+  } 
+
+  function displaySearchResults(results, store, searchTerm) {
+    var searchResults = document.getElementById('search-results');
+
+    if (results.length) {
+      var appendString = '';
+
+      for (var i = 0; i < results.length; i++) {
+        var item = store[results[i].ref];
+        if (isBlank(item.title)) {
+            item.title = "No title";
+        }
+        appendString += '<li><a href="' + item.url + '"><h3>' + item.title + '</h3></a>';
+	      var indx = item.content.indexOf(searchTerm);
+	      var start_pos = Math.max(0, indx - 75);
```

Comment:
better to write code in one style, if you use camelCase - always use it, if you use lisp_case - always use it